packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts

```
@@ -211,6 +223,187 @@ export function angular(options?: PluginOptions): Plugin[] {
+    // JIT mode
+    if (jit) {
+      const result = analogJitTransform(code, id);
+      return { code: result.code, map: result.map };
+    }
+
+    // Inline external templateUrl/styleUrl(s) into the source before compilation
+    // using OXC parser for precise AST-based rewriting.
+    code = inlineResourceUrls(code, id);
+
+    // Pre-resolve inline styles that need preprocessing (SCSS/Sass/Less)
+    let resolvedStyles: Map<string, string> | undefined;
```

ESLint

[error] 'resolvedStyles' is always 'undefined' because it's never assigned.

(no-unassigned-vars)